test/bus_test.dart

```diff
@@ -561,5 +610,36 @@ void main() {
       final simResult = SimCompare.iverilogVector(gtm, vectors);
       expect(simResult, equals(true));
     });
+
+    test('selectFrom and selectIndex', () async {
+      final gtm = SelectTestModule(Logic(width: 8), Logic(width: 8),
+          Logic(width: 8), Logic(width: (log(8) / log(2)).ceil()));
+      await gtm.build();
+      final vectors = [
+        Vector({'a1': 1, 'a2': 2, 'a3': 3, 'b': 1},
+            {'selectIndexValue': 2, 'selectFromValue': 2}),
+        Vector({'a1': 1, 'a2': 2, 'a3': 3, 'b': 0},
+            {'selectIndexValue': 1, 'selectFromValue': 1}),
+        Vector({'a1': 1, 'a2': 2, 'a3': 3, 'b': 2},
+            {'selectIndexValue': 3, 'selectFromValue': 3})
+      ];
+      await SimCompare.checkFunctionalVector(gtm, vectors);
+      final simResult = SimCompare.iverilogVector(gtm, vectors);
+      expect(simResult, equals(true));
+    });
+
+    test('selectFrom with Index', () async {
```

Comment:
nit: name of test, this one is for default right? not sure what "with Index" means